client/guidestours/style.scss

```diff
@@ -1,7 +1,33 @@
 .guidestours__step {
 	position: fixed;
-	width: 400px;
-	z-index: z-index( 'root', '.web-preview' );
+	width: 410px;
+	z-index: z-index( 'root', '.guidestours__step' );
+	border: 1px solid lighten( $gray, 10 );
+	box-shadow: 0px 10px 23px 0px rgba( 0, 0, 0, 0.50 );
+	border-radius: 4px;
+	padding-top: 19px;
+	font-size: 14px;
+
+	p {
```

Comment:
@lsinger sorry, missed this reply. The rule of thumb should be to always have a class. We use `a` in a few cases where we don't control what may get rendered.

In general generic selectors are more expensive and read from right to left (so it selects all "a" in the page, then figures out which are children of your namespaced class).

`.button` is a no brainer, because Button is a component and that's how we modify nested components `.components .nested-component {}` which has the benefit of added specificity and less chance of breakage. It also signals to the reader that these are nested components while `.component button {}` does not.